.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 
 # Docker environment.
 defaults: &defaults
-  working_directory: /go/src/github.com/cisco-app-networking/nsm-nse
```

Comment:
I don't think we need to update this path here and below in this file. This file can stay unchanged.

Reply:
At this point any change on circle CI is additional pain, this will go into the develop_jenkins branch and so no circle CI changes will be tested